app/controllers/IdentityVerification.scala

```diff
@@ -76,15 +76,19 @@ class IdentityVerification @Inject()(
   }
 
   def success(journeyId: Option[String]): Action[AnyContent] = ggAction.async { implicit request =>
-    journeyId.fold(Future.successful(Unauthorized(errorHandler.internalServerErrorTemplate))) { id =>
-      identityVerificationConnector.verifySuccess(id).flatMap {
-        case true =>
-          identityVerificationService.continue(id, request.userDetails).map {
-            case Some(obj) => identityVerificationService.someCase(obj)
-            case None      => identityVerificationService.noneCase
-          }
-        case false => Future.successful(Unauthorized(errorHandler.internalServerErrorTemplate))
+    if (config.ivEnabled) {
+      journeyId.fold(Future.successful(Unauthorized(errorHandler.internalServerErrorTemplate))) { id =>
+        identityVerificationConnector.verifySuccess(id).flatMap {
+          case true =>
+            identityVerificationService.continue(id, request.userDetails).map {
+              case Some(obj) => identityVerificationService.someCase(obj)
+              case None      => identityVerificationService.noneCase
+            }
+          case false => Future.successful(Unauthorized(errorHandler.internalServerErrorTemplate))
+        }
       }
+    } else {
+      Future.successful(Redirect(controllers.registration.routes.RegistrationController.success(1L)))
```

Comment:
1L is a success code?

Reply:
no, it's just some person id gets displayed on a confirmation screen. can't dig that out of nowhere since this is iv success screen but we have switched IV off...